docs/admin-guide/load-config.md

```diff
@@ -1,125 +1,93 @@
-	- *rhocall_bed* Path to bed file to view alignments (Reference)[https://github.com/dnil/rhocall]
-	- *rhocall_wig* Path to wig file to view alignments (Reference)[https://github.com/dnil/rhocall]
-	- *upd_regions_bed* Path to bed file to view alignments (Reference)[https://github.com/bjhall/upd]
-	- *upd_sites_bed* Path to bed file to view alignments (Reference)[https://github.com/bjhall/upd]
-	- *tiddit_coverage_wig* Path to wig file to view alignments (Reference)[https://github.com/SciLifeLab/TIDDIT]
-    - *vcf2cytosure* Path to CGH file to allow download per individual
-    - *tumor_type* Type of tumor
-    - *tissue_type* What tissue the sample originates from
-    - *tmb* Tumor mutational burden
-    - *msi* Microsatellite instability
-    - *tumor_purity* Purity of tumor sample
-
-- **vcf_snv** path to snv vcf file
-- **vcf_sv**
-- **vcf_snv_research** path to vcf file with all variants
-- **vcf_sv_research**
-- **vcf_cancer**
-- **vcf_cancer_research**
-- **madeline** path to a madeline pedigree file in xml format
-- **peddy_ped** path to a [peddy][peddy] ped file with an analysis of the pedigree based on variant information
-- **peddy_check** path to a [peddy][peddy] ped check file
-- **peddy_sex** path to a [peddy][peddy] ped sex check file
-- **multiqc** path to a [multiqc][multiqc] report with arbitrary information
-- **cnv_report** path to the CNV report file
-- **coverage_qc_report** path to static coverage and qc report file
-- **gene_fusion_report** path to a static file containing a gene fusion report produced by [Arriba][arriba]. Generated from default clinical data.
-- **gene_fusion_report_research** path to a static file containing a gene fusion report produced by [Arriba][arriba]. Generated from research data.
-- **default_gene_panels** list of default gene panels. Variants from the genes in the gene panels specified will be shown when opening the case in scout
-- **gene_panels** list of gene panels. This will specify what panels the case has been run with
-- **rank model version** which rank model that was used when scoring the variants
-- **SV rank model version** the SV rank model version used when scoring SV variants
-- **rank_score_treshold** only include variants with a rank score above this treshold
-- **analysis_date** time for analysis in datetime format. Defaults to time of uploading
-- **human_genome_build** what genome version was used.
-
-### Minimal config
+# The Load Config File
+
+A case and its individuals (or samples, in cancer track) can be uploaded into Scout using specially formatted .yaml config files. Config files contain information about the analysis, panels used, path to VCF and eventual alignment files and more. The following template illustrates the basic structure of the .yaml config file:
+
+The format is [Yaml][yaml].
+
+Example configuration files are found here: `<scout root dir>/scout/demo/643594.config.yaml`. [Link](../../scout/demo/643594.config.yaml).
```

Comment:
The link to this file doesn't work. If you run the preview:
- `mkdocs build`
-` mkdocs serve`

It looks like this:

![image](https://user-images.githubusercontent.com/28093618/118081174-25d67500-b3bb-11eb-967d-b7af56cc8632.png)

Reply:
It is perhaps best to link to the repository. AFAIK only the docs get copied to the GitHub IO doc site area on deploy.

Reply:
> It is perhaps best to link to the repository. AFAIK only the docs get copied to the GitHub IO doc site area on deploy.

Good idea! Implemented now: `08c1904`.